client/main.js

```diff
@@ -101,6 +91,7 @@ class ReputationMiningClient {
     const tx = await this.ganacheWallet.sendTransaction(patriciaTreeDeployTx);
     this.reputationTree = new ethers.Contract(ethers.utils.getContractAddress(tx), PatriciaTreeJSON.abi, this.ganacheWallet);
     this.nReputations = 0;
+    this.reputations = {};
```

Comment:
This secret key, is it ever going to change? Is it ok for it to be hardcoded here just like that?
https://github.com/JoinColony/colonyNetwork/pull/223/files#diff-096103d8490b885f9d5037baea6aea80R16

Reply:
I can't see why it would ever change. This is the key used by the internal ganache instance used to host the Patricia Tree. It's never used for anything outward facing.